web-console/4.0-designs/status/status.md

```diff
@@ -47,3 +47,13 @@ Note: Icons should never be animated in a list view.
 * [Color](https://www.patternfly.org/styles/color-palette/)
   * **Black**: pf-black #030303
   * **Blue**: pf-blue-400 #0088CE (for links only)
+
+## Status Popovers
+
+If additional information about a status is available, a popover can be used to provide context and up to two suggested actions. Statuses with popovers should be styled in the same way as inline links to indicate interactivity.
+
+![Status popover in table](img/status-popover-table.png)
+
+Status popovers can include links, progress bars, and dynamically-updated content. Up to two contextual actions can be included in the footer of the popover.
```

Comment:
should we clarify that there can also be actions included in the body of the popup?

Reply:
I'm not quite sure about this - do you have an example where this would be helpful?

I'd worry somewhat about users confusing hyperlinks with (what should be) inline link buttons for actions if we mix both inside the body. I'm thinking restricting actions to the footer might standardize their interaction and discourage us from putting more than we should into these popovers.

Reply:
I made a small tweak to add an external link icon whenever blue text in the body or footer would bring the user to another site. Blue text within the body would link to other internal pages, and blue "action" text within the footer would behave like... actions.

Reply:
+1 on the external link icon. That definitely helps separate things in the body.
I agree we don't want to over run these with actions.
Thinking about this from an accessibility point of view an action could be more helpful in the context of the content than as a separate action in the footer.

Reply:
I softened the wording to allow for actions within the body "when it makes sense to do so" to accommodate potential use cases that we can't predict right now. We can make our guidance more prescriptive again if this leniency starts making things too inconsistent.